searches/linear_search.py

```diff
@@ -27,9 +27,9 @@ def linear_search(sequence: list, target: int) -> int:
     >>> linear_search([0, 5, 7, 10, 15], 6)
     -1
     """
-    for index, item in enumerate(sequence):
-        if item == target:
-            return index
+    for ind, itm in enumerate(sequence):
+        if itm == target:
+            return ind
     return -1
```

Comment:
This is trivial and I think this doesn't improve our code

Reply:
Please let me know if you've other thoughts. Feel free to make other improvements!